freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/UserSyncService.java

```diff
@@ -72,6 +80,8 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(UserSyncService.class);
 
+    private static final Integer BATCH_ADD_REMOVE_MEMBER_PARTITION_SIZE = 50;
```

Comment:
shouldn't this be configurable via environment?

Reply:
i was not sure about this, i am afraid if someone uses too high value, it will cause issues and it requires lot of testing also. but if you think it is worth to make it configurable, i'll make that change

Reply:
I can't say I have seen we changed something like this in prod as a hotfix, but if we would like to run it with different numbers in dev environments to test it, then we could apply it easily without affecting prod. So yes, I think it worth moving out.